### ISSUE TITLE
Make it less likely for users to accidentally miss the /WCSim/Construct line in novis.mac/vis.mac

### ISSUE THREAD

**Issue by eosullivan** [WCSim/WCSim#79]
Thanks to @wilking and @cvilelasbu for bringing this issue to our attention. Currently, the /WCSim/Construct line is buried after a bunch of detector configurations, which makes it easy to miss. If a user wishes to change the detector configuration in the macro, but doesn't uncomment out the  /WCSim/Construct line, the new detector configuration will not be properly built. We should either find a way of automatically implementing this line if a new detector configuration is called, or at the very least should find a way to make this line more visible.

**Comment by wilking** [comment]
Thank you for documenting this! One possible solution would be to always require a user to specify a geometry, rather than have a default geometry (which doesn't behave as intended, anyway, since the PMT charge response is often overwritten later). There was some concern that having the detector geometry be built twice is wasteful, which I agree with, so perhaps the code could be rewritten to never construct a geometry by default, and only construct a single geometry after it is specified in the input macro file.

**Comment by cwwalter** [comment]
I would still like a default geometry built.  I want the code to work by default if you don't know how to make changes.  If we can't make this work in one-line (which I think may be the case if I remember correctly) I think there are a few easy things we can do to fix this.  It never used to be a problem and I think the reason it is a problem now is that there is such a separation between the lines.  So, I suggest:

1) Editing the file so that at the very top where it says to uncomment new geometries below here, it also says the construct line must be uncommented.

2) At least in vis.mac (but not novis) there are other options before the construct line like the collection efficiency and the QE method. This should be changed.

3) Probably most importantly.  Every possible option should not be listed here.  Just a few common ones.  People are adding every option to this file when they put code in.  In particular all of the LBNE options should be removed.  If necessary we could make another vis file with possible ones in it for people to look at.

The effect of these changes should be to have a very short section where the first line and the construct line are all visible next to each other.

**Comment by cwwalter** [comment]
@eosullivan Can you also add a new issue about including options by including other .mac files like we talked about before?  That is another way to approach this.  If someone wanted another geometry they could do the equivalent of:

#include mygeom.mac

**Comment by tdealtry** [comment]
Could someone please clarify - does 'new geometry' mean 'a geometry that isn't SuperK'?

**Comment by cwwalter** [comment]
@tdealtry In practice, yes.
  
But, really what it means is that you have forced WCSim to build a new geometry.  You can always do this (And it could in principle be the same as what you already built).

The main point is that one geometry is built by default when you start.  That is hard coded in the C++ code.  Currently it is set to build SK.  So if you want something else you need to force a new geometry.  After we decide on a a real HK configuration we may set it to that.

Basically we set it to whatever the 'default' or 'most common' configuration is. Aside from the fact that we want there to be a sensible set of defaults you always need some geometry defined when you build a G4 program.

Does that make sense?

**Comment by tdealtry** [comment]
Yeah that makes sense. Thanks @cwwalter 

Can I recommend adding a line to the .mac file along the lines of:

#/WCSim/Construct # This must be uncommented in order to construct a new detector configuration. 
#In practice, this means whenever a /WCSim/WCgeom option is given, this line must follow (uncommented).
